apps/web/src/ui/pool/ClaimableFeesTab.tsx

```diff
@@ -0,0 +1,183 @@
+'use client'
+
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  Container,
+  DataTable,
+} from '@sushiswap/ui'
+import type { ColumnDef, PaginationState } from '@tanstack/react-table'
+import React, { type FC, useMemo, useState } from 'react'
+import { useAllPrices } from 'src/lib/hooks/react-query'
+import { useConcentratedLiquidityPositions } from 'src/lib/wagmi/hooks/positions/hooks/useConcentratedLiquidityPositions'
+import type { ConcentratedLiquidityPositionWithV3Pool } from 'src/lib/wagmi/hooks/positions/types'
+import { type SushiSwapV3ChainId, SushiSwapV3ChainIds } from 'sushi/config'
+import { Amount, type Type } from 'sushi/currency'
+import { useAccount } from 'wagmi'
+import {
+  FEES_ACTION_COLUMN,
+  FEES_AMOUNT_COLUMN,
+  FEES_CHAIN_COLUMN,
+} from './columns'
+
+export type ClaimableFees = {
+  chainId: SushiSwapV3ChainId
+  feeAmounts: Record<string, Amount<Type>>
+  feeAmountsUSD: Record<string, number>
+  totalFeesUSD: number
+  positions: ConcentratedLiquidityPositionWithV3Pool[]
+}
+
+const COLUMNS = [
+  FEES_CHAIN_COLUMN,
+  FEES_AMOUNT_COLUMN,
+  FEES_ACTION_COLUMN,
+] satisfies ColumnDef<ClaimableFees, unknown>[]
+
+export const ClaimableFeesTab: FC = () => {
+  const { address, isConnecting } = useAccount()
+  const { data: _data, isInitialLoading: isPositionsLoading } =
+    useConcentratedLiquidityPositions({
+      account: address,
+      chainIds: SushiSwapV3ChainIds,
+    })
+
+  const [paginationState, setPaginationState] = useState<PaginationState>({
+    pageIndex: 0,
+    pageSize: 10,
+  })
+
+  const { data: prices, isLoading: isPricesLoading } = useAllPrices()
+
+  const isLoading = isPositionsLoading || isPricesLoading
+
+  const data = useMemo(() => {
```

Comment:
I'd probably extract these into a hook, just to keep the component file a bit shorter. nitpick tho